tests/test_bot.py

```diff
@@ -385,6 +400,97 @@ def test(_, url, data, *args, **kwargs):
                                                switch_pm_text='switch pm',
                                                switch_pm_parameter='start_pm')
 
+    def test_answer_inline_query_current_offset_error(self, bot, inline_results):
```

Comment:
I don't think I understand you here

Reply:
https://docs.pytest.org/en/latest/parametrize.html#pytest-mark-parametrize

This generates new tests given a bunch of parameter combinations so that you don't need to make multiple assertions in a single test.

Reply:
Here's an example for classs-based test method parametrization:

```
class TestCase:

    @pytest.mark.parametrize("test_input,expected", [
    ("3+5", 8),
    ("2+4", 6),
    ("6*9", 42),  # fails
    ])
    def test_1(self, test_input, expected):
        assert eval(test_input) == expected
```

Reply:
I know that ;) I just don't see what you'd want to parametrize on this test